microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md

```diff
@@ -184,7 +186,7 @@ This article gives you a lot to learn. But now you have a solid foundation in th
 <!-- ------------------------------ -->
```

Comment:
just above this section https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/evaluate-performance/?branch=pr-en-us-2814#analyze-the-optimized-version
seems an oddly placed paragraph:
"This article gives you a lot to learn. But now you have a solid foundation in the basic workflow for analyzing runtime performance.  Good job."
Please check flow; is this the best place for that paragraph?